notebook/analysis/simulation_results.ipynb

```diff
@@ -16,103 +25,508 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<ggplot: (8761974809853)>"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "plot = ggplot(sepsis_metrics[sepsis_metrics['supervised'] == 'three_layer_net'], aes(x='balanced_accuracy')) \n",
+    "plot += geom_density()\n",
+    "plot += ggtitle('Simulated Sepsis Accuracy vs Sample Count')\n",
```

Comment:
This title shouldn't be "vs Sample Count", should it? Maybe something like "Distribution of simulated sepsis accuracy"?

Reply:
Good catch!